src/server/models/db/migrations/20221111151312_cascade_deletion_protection_and_new_event_types/migration.sql

```diff
@@ -16,7 +16,7 @@ COMMIT;
 
 -- AlterEnum
 BEGIN;
-CREATE TYPE "Status_new" AS ENUM ('NEW', 'OUT_WITH_PROVIDER', 'EDITED', 'PUBLISHED', 'UNPUBLISHED', 'CHECK_ANNUAL_REVIEW', 'ANNUAL_REVIEW_OVERDUE');
+CREATE TYPE "Status_new" AS ENUM ('NEW', 'OUT_WITH_PROVIDER', 'EDITED', 'PUBLISHED', 'UNPUBLISHED', 'CHECK_ANNUAL_REVIEW', 'ANNUAL_REVIEW_OVERDUE', 'OUT_FOR_ANNUAL_REVIEW');
```

Comment:
OUT_FOR_ANNUAL_REVIEW shouldn't be a (new) status type

Reply:
Hmmm, in that case do you know what would trigger this status combination?

🟢 Live/ 🟡 out for annual review:

Unpublish

Or is this something I shouldn't worry about for this ticket?

Reply:
hey @RichardBray , I think the following logic is needed here:

* UI displays OUT_FOR_ANNUAL_REVIEW when ListItem.status = OUT_WITH_PROVIDER and ListItem.isAnnualReview = true
* UI displays EDITS_REQUESTED when when ListItem.status = OUT_WITH_PROVIDER and ListItem.isAnnualReview = false

For both these scenarios I also think we should display the Request Edits option so Post can send another link if a user has accidentally used it, browser crashes or they didn't submit it and the link won't work again, for example.

Reply:
Hey @alisalaman-cyb looks like that logic is already in place for the activity status label in the `getActivityStatus()`  from `summary.helpers.ts`. I guess similar logic needs to be added to the radio buttons 🤔